Readme.md

```diff
@@ -74,14 +76,14 @@ Note: You may supply the baseURL (your website's root) via the environment varia
 
 ### Usage
 
-Point your users to `/auth/{twitter,facebook,github,google}` to have them login (or signup) via the specified module.
+Point your users to `/auth/{twitter,facebook,github,google,dribble,weibo}` to have them login (or signup) via the specified module.
 After that, Auth-Passport completely takes over and redirects the users according to the OAuth(2) flow.
 
 Also You can use `/auth/login` to login on local user collection but it has to be POST method.
 
 ### Usage in Mobile Apps
 
-Auth-Passport was built with usage in mobile Apps in mind. From inside your mobile app, open a browser and point the user to your website's `/auth/{login,twitter,facebook,github}` endpoint. From there, Auth-Passport will take over and guide (i.e. redirect) your user through the different steps needed for each provider, until the user has authorized your app and logged in successfully.
```

Comment:
could you please readd the local login part? /auth/login
I just noticed, that it also misses in the local login in the usage. Would be great if you could include login there too